frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart

```diff
@@ -29,82 +27,6 @@ class _LandingPageNewState extends State<LandingPageNew> {
       super.dispose();
     }*/
 
-  var state = [
-    'Andhra Pradesh',
-    'Arunachal Pradesh',
-    'Assam',
-    'Bihar',
-    'Chhattisgarh',
-    'Goa',
-    'Gujarat',
-    'Haryana',
-    'Himachal Pradesh',
-    'Jharkhand',
-    'Karnataka',
-    'Kerala',
-    'Madhya Pradesh',
-    'Maharashtra',
-    'Manipur',
-    'Meghalaya',
-    'Mizoram',
-    'Nagaland',
-    'Odisha',
-    'Punjab',
```

Comment:
Why state is removing ? @shubhamdevgupta

Reply:
this is static image we created for mobile now we will take this from asserts/json file

Reply:
Okay @shubhamdevgupta